pepper-apis/src/test/java/org/broadinstitute/ddp/route/CreateActivityInstanceRouteTest.java

```diff
@@ -189,33 +175,4 @@ public void testCreateActivityShouldFailBecauseTooManyInstances() {
                 .contentType(ContentType.JSON)
                 .body("code", Matchers.equalTo(ErrorCodes.TOO_MANY_INSTANCES));
     }
-
-    @Test
-    public void testCreateActivityFailsWhenPexExpressionNotMet() {
```

Comment:
These seem like important tests.  Where have they gone?  Or why are they no longer needed?

Reply:
This was back when we thought Activity's should be the final say on wether themselves should be created. We've since moved this functionality into Events